src/itest/java/uk/gov/companieshouse/pscdataapi/steps/PscDataSteps.java

```diff
@@ -75,16 +95,16 @@ public class PscDataSteps {
     private final String contextId = "5234234234";
 
     @Before
-    public void dbCleanUp(){
+    public void dbCleanUp() {
         if (!mongoDBContainer.isRunning()) {
             mongoDBContainer.start();
         }
         companyPscRepository.deleteAll();
-        MockitoAnnotations.initMocks(this);
+        MockitoAnnotations.openMocks(this);
```

Comment:
This has new method returns an instance if `AutoCloseable`. According to the docs, the `@Before` method needs to be matched with an `@After` method that closes the object. e.g.
```
@BeforeEach public void openMocks() {
     closeable = MockitoAnnotations.openMocks(this);
}

@AfterEach public void releaseMocks() throws Exception {
    closeable.close();
} 
```
See [JavaDoc](https://javadoc.io/static/org.mockito/mockito-core/5.8.0/org/mockito/MockitoAnnotations.html) and [SO post](https://stackoverflow.com/questions/75970985/mockitoannotationns-openmocks-autoclosable)